rootfs/ingest/python/qserv/http.py

```diff
@@ -153,10 +160,30 @@ def _authenticate(self, auth_path: Optional[str]) -> str:
             authKey = getpass.getpass()
         return authKey
 
-    def get(self, url: str,
-            payload: Dict[str, Any] = dict(),
-            auth: bool = True,
-            timeout: Optional[int] = None) -> Dict:
+    def get(
+        self, url: str, payload: Dict[str, Any] = dict(), auth: bool = True, timeout: Optional[int] = None
+    ) -> Dict:
+        """Send a GET query to an http(s) URL.
+
+        Parameters
+        ----------
+        url : `str`
+            Http(s) URL
+        payload : `Optional[Dict[str, Any]`
```

Comment:
Need second closing bracket. 

Complex type formatting is done differently, You need something like:

    payload : `Dict` [`str`, `Any`], optional

see   https://developer.lsst.io/python/numpydoc.html#sequence-types

Reply:
@andy-slac writing the correct docstring format is complicated and time-consuming. I do not understand why LSST does not use a tool to automatically generate dosctring, that the developer could then fill with useful information. I can easilly do that in VS code plugin, but unfortunately the docstring does not fit the LSST standards:
```
    def myfunc(
        self, url: str, payload: Dict[str, Any] = dict(), auth: bool = True, timeout: Optional[int] = None
    ) -> Dict:
        """_summary_

        Args:
            url (str): _description_
            payload (Dict[str, Any], optional): _description_. Defaults to dict().
            auth (bool, optional): _description_. Defaults to True.
            timeout (Optional[int], optional): _description_. Defaults to None.

        Returns:
            Dict: _description_
        """
```

Reply:
@fjammes, reviewing a pull request is time consuming. Please try to make sure that your code follows LSST conventions when you send it for review.